base.config

```diff
@@ -62,3 +62,11 @@ CONFIG_KERNEL_GZIP=y
 
 ## Fix the path for Gentoo.
 CONFIG_MODPROBE_PATH="/sbin/modprobe"
+
+## Enable apparmor. Fedora only provides SELinux support.
+## https://bugs.gentoo.org/834918
+CONFIG_SECURITY_APPARMOR=y
+CONFIG_SECURITY_APPARMOR_HASH=y
+CONFIG_SECURITY_APPARMOR_HASH_DEFAULT=y
+# CONFIG_SECURITY_APPARMOR_DEBUG is not set
+# CONFIG_DEFAULT_SECURITY_APPARMOR is not set
```

Comment:
Do you need these two?

Reply:
These only become visible after enabling `CONFIG_SECURITY_APPARMOR` but you should leave them disabled. I don't claim to understand how gentoo-kernel / gentoo-kernel-bin work but I saw other commented options in this file and assumed each option needs 0 or 1. But they are disabled by default so they can be left out if explicit "configure option" isn't required here.

Reply:
Well, I think either way is fine, as long as the end result is correct. I leave the final choice to you.